src/localization/en.js

```diff
@@ -188,7 +200,65 @@ export default {
       },
     },
 
-    currentExpenses: {},
+    currentExpenses: {
+      header:                          'Current Household Expenses',
+      unreimbursedNonMedicalChildCare: {
+        sectionHeading:    'Reasonable Unreimbursed Non-Medical Child(ren) Care',
+        subheading:        'A "child" is a person 12 or younger. Don\'t include amounts that are paid for by other benefit programs.',
+        columnExpenseType: 'Expense',
+        childDirectCare:   {
+          label: 'Direct care costs',
+          hint:  'How much do you pay for child care out of pocket?',
+        },
+        childBeforeAndAfterSchoolCare: {
+          label: 'Before- and after-school care',
+          hint:  'How much do you pay for child care for times before or after school?',
+        },
+        childTransportation: {
+          label: 'Transportation costs',
+          hint:  'How much do you pay for transportation?',
+        },
+        childOtherCare: {
+          label: 'Other care',
+          hint:  'How much do you pay for other child care?',
+        },
+        earnedBecauseOfChildCare: 'Does child care allow you to make additional income?',
+        ifYes:                    'Income made possible by childcare expenses',
```

Comment:
How about putting these two in a sub-object called `earnedBecauseOfChildCare` (you could call the question itself something else) and also including a hint, like the other rows?

Reply:
What's the reasoning behind that design decision? Right now it's just two questions - one of them just reveals the other, like many questions - and I don't see why we'd put two questions into a new object of their own.

I can see how this would be a sub-section of a form in some ways (and one of the unfinished discussions is about if this should actually be a sub-section). So far we haven't created objects for sub-sections. We could possibly talk about breaking out all sub-sections into objects. I'm not sure how much more nesting we want. What are the pros and cons?

Reply:
OK, I was thinking of the radio button and income row as one unit, since the radio is just there to draw attention to / introduce the income row. If we are going to treat it as its own thing though, then I think the income row about earned income because of child care should be called something more descriptive.

We could use `doEarnBecauseOfChildCare` for the radio button and `earnedBecauseOfChildCare` for the amount, maybe?

Reply:
Yeah, that makes sense. Something like that sounds good.